manim/cli/render/commands.py

```diff
@@ -162,18 +162,19 @@ def __repr__(self):
         except Exception:
             logger.debug(f"Something went wrong: {warn_prompt}")
 
-        try:
-            stable = req_info.json()["info"]["version"]
-
-            if stable != __version__:
-                console.print(
-                    f"You are using manim version [red]v{__version__}[/red], but version [green]v{stable}[/green] is available."
-                )
-                console.print(
-                    "You should consider upgrading via [yellow]pip install -U manim[/yellow]"
-                )
-        except json.JSONDecodeError:
-            logger.debug(warn_prompt)
-            logger.debug(f"Error decoding JSON from {manim_info_url}")
+        if req_info.ok:
```

Comment:
Since this is only checking that the code is < 400, don't we want to explicitly check for a 200/success instead of redirects and other things?
```suggestion
        if req_info.status_code == requests.codes['ok']:
```

Reply:
I actually considered doing it like this:
```python
if req_info.status_code == 200:
```
I thought ```request.ok``` does the same, but I should have read the documentation. 
Sorry about that.
I'll fix it right away

Reply:
Or simply move the below code to the previous catch block so that if it error's out this won't run at all.

Reply:
Yes of course, I completely missed that. This is similar to "ask forgiveness not permission", right ? Or is it more like a taste/style thing ?